x/stakeibc/keeper/msg_server_register_host_zone_test.go

```diff
@@ -0,0 +1,85 @@
+package keeper_test
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	ibctesting "github.com/cosmos/ibc-go/v3/testing"
+	_ "github.com/stretchr/testify/suite"
+
+	recordstypes "github.com/Stride-Labs/stride/x/records/types"
+	recordtypes "github.com/Stride-Labs/stride/x/records/types"
+	stakeibc "github.com/Stride-Labs/stride/x/stakeibc/types"
+)
+
+type RegisterHostZoneTestCase struct {
+	validMsg stakeibc.MsgRegisterHostZone
+}
+
+func (s *KeeperTestSuite) SetupRegisterHostZone() RegisterHostZoneTestCase {
+	s.CreateTransferChannel(HostChainId)
+
+	s.App.StakeibcKeeper.SetEpochTracker(s.Ctx(), stakeibc.EpochTracker{
+		EpochIdentifier:    "day",
+		EpochNumber:        3,
+		NextEpochStartTime: uint64(2661750006000000000), // arbitrary time in the future, year 2056 I believe
```

Comment:
Is this used? If not, can we remove it?

If we need it, let's do time.Now() + Duration